app/src/main/java/com/madonasyombua/growwithgoogleteamproject/models/User.java

```diff
@@ -8,9 +8,9 @@
     Just a dummy model test the fragment
  */
 public class User {
-    private String name, followers, following, projects;
-    private String email, location, phone, website, intro;
-    private String status;
+    private String name, password, followers, following, projects,
```

Comment:
User should not have a password field because fire base auth manages that

Reply:
@CleverChuk  but we should have a password field, FirebaseAuth need it as entry, the only thing we can do is not encrypting it.

Reply:
@CleverChuk I used to pass User data as params

Reply:
This might be an older version. I wanted some inputs in the Nav drawer. I believe @MahaSoua has a recent . This is just for review not to be merged.

Reply:
@MaherSoua Would that user object be discarded after use?
@Madonahs Got it

Reply:
@CleverChuk  I think maybe you should review the new Develop Branch 👍  this was old. Might have errors.

Reply:
@CleverChuk we can stored, we have choice it depends on we would do after registration or login.